Podfile

```diff
@@ -1,10 +1,10 @@
 # Uncomment the next line to define a global platform for your project
-# platform :ios, '9.0'
+platform :ios, '9.0'
 
 target 'JWPlayer-SDK-iOS-Demo' do
   # Comment the next line if you don't want to use dynamic frameworks
   use_frameworks!
 
   # Pods for JWPlayer-SDK-iOS-Demo
-	pod 'JWPlayer-SDK'
+  pod 'JWPlayer-SDK', '3.16.2'
```

Comment:
shouldn't the version string be '~> 3.16.2' so it will always find the latest iOS SDK?

Reply:
In this instance, I don't think so.

`~>` is great for dependency management to model a rule like "the latest version that will not break anything", but it turns out that this is mostly used as an Issue Reproduction app, which means switching back and forth to compare specific versions. The version the customer wants is running is rarely the latest. 

It also lets us instruct less technical customers to just change the version number string to the one they/we want.